### ISSUE TITLE
Feature Request: Qwen3 Embedding (along with Mistral and any others easily supportable by current architecture)

### ISSUE THREAD

**Issue by sempervictus** [EricLBuehler/candle-vllm#211]
Attempted to spin up `Salesforce/SFR-Embedding-Mistral`, `BAAI/llm-embedder`, and `Qwen/Qwen3-Embedding-8B` all with different failures. Upon loading. Most modern workflows require RAG and accelerated 4KD generation can make a significant difference in workflow efficacy. I know this is an outstanding issue in mistralrs as well which is focused on multimodal while this library seems to be text-focused.

**Comment by guoqingbao** [comment]
> Attempted to spin up `Salesforce/SFR-Embedding-Mistral`, `BAAI/llm-embedder`, and `Qwen/Qwen3-Embedding-8B` all with different failures. Upon loading. Most modern workflows require RAG and accelerated 4KD generation can make a significant difference in workflow efficacy. I know this is an outstanding issue in mistralrs as well which is focused on multimodal while this library seems to be text-focused.

Your observation is right, we haven't expose raw logits for embedder usage. Perhaps we can add one because the embedding model has similar structure with its corresponding text model.

**Comment by sempervictus** [comment]
Thank you @guoqingbao - that Qwen3 8B is probably the most effective target out of the supported model types.
